crates/bevy_math/src/bounding/bounded2d.rs

```diff
@@ -0,0 +1,331 @@
+use super::BoundingVolume;
+use crate::prelude::Vec2;
+
+/// A trait with methods that return 2D bounded volumes for a shape
+#[doc(alias = "BoundingRectangle")]
+pub trait Bounded2d {
```

Comment:
I think you accidentally added the doc alias to the wrong struct? This should be on `Aabb2d`